client/wordpress-com.js

```diff
@@ -314,54 +314,44 @@ if ( config.isEnabled( 'reader' ) ) {
 	} );
 }
 
-if ( config.isEnabled( 'vip' ) ) {
-	sections.push( {
-		name: 'vip',
-		paths: [ '/vip', '/vip/deploys', '/vip/billing', '/vip/support', '/vip/backups', '/vip/logs' ],
-		module: 'vip',
-		secondary: true
-	} );
-}
+sections.push( {
```

Comment:
Thanks for letting me know. I'll open a follow up PR once this one gets merged ;)